packages/backend/tests/env.js

```diff
@@ -0,0 +1,20 @@
+process.env.FS_BACKEND = "http://localhost:3000";
```

Comment:
This file is just a copy of the example `.env` file. I don't know how this is normally done but I want to the tests to run the same regardless of the environment.